src/main/java/com/crowdin/cli/commands/actions/StatusAction.java

```diff
@@ -92,6 +95,15 @@ public void act(Outputter out, ProjectProperties pb, ProjectClient client) {
                 progresses.forEach(pr -> out.println(String.format(RESOURCE_BUNDLE.getString("message.item_list_with_percents"),
                     pr.getLanguageId(), pr.getApprovalProgress())));
             }
+            if (failedIfIncomplete) {
+                progresses.stream()
+                    .filter(p -> p.getApprovalProgress() <= 100)
```

Comment:
Also, we should have the `getTranslationProgress` check for the `crowdin status translation` command

Reply:
This behavior is incorrect.

For the `status` and `status proofreading` we should check the `getApprovalProgress`

And for the `status translation` we should check `getTranslationProgress`

Not both at the same time.